docfx.sln

```diff
@@ -16,6 +16,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tools", "tools", "{7C85AA4C
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CreateJsonSchema", "tools\CreateJsonSchema\CreateJsonSchema.csproj", "{A899DBCD-C7A3-4372-AA0C-5FA542759BBB}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ImportTemplateResource", "tools\ImportTemplateResource\ImportTemplateResource.csproj", "{B52B4C8B-DFF2-43CF-B7B5-3DA058CCD6F0}"
```

Comment:
since this is mostly an ad-hoc tool, can we put it somewhere else like a personal github repo?

Reply:
i'd like to keep it in tools in case we need run it again and again before new template is implemented, and also we can added it to CI job